src/Upgrade/Application/Strategy/ReleaseApp/Mapper/PackageCollectionMapperInterface.php

```diff
@@ -28,6 +28,13 @@ public function mapModuleCollectionToPackageCollection(ModuleDtoCollection $modu
      */
     public function getRequiredPackages(PackageCollection $packageCollection): PackageCollection;
 
+    /**
+     * @param \Upgrade\Domain\Entity\Collection\PackageCollection $packageCollection
+     *
+     * @return \Upgrade\Domain\Entity\Collection\PackageCollection
+     */
+    public function getUpdatedPackages(PackageCollection $packageCollection): PackageCollection;
```

Comment:
Can be renamed to the `getSubPackages` or `filterSubPackages`. In the code it is used as a `subPackages`, so it is logical to keep all the names aligned.